connectivity/cycles.py

```diff
@@ -2,7 +2,7 @@
 import networkx as nx
 
 zipCode = '02138'

```

Comment:
Oh good catch. It's the Cambridge MA zip code. I don't believe either of these scripts are in active use ATM, but I'll need to fixup the WHERE clause in your other comment anyways.